### ISSUE TITLE
Update readme for wp-rocket-no-cache

### ISSUE THREAD

**Issue by socialpreneur** [wp-media/wp-rocket-helpers#39]
Could you update readme for the following no cache plugin?
https://github.com/wp-media/wp-rocket-helpers/tree/master/cache/wp-rocket-no-cache

I think current code works with any newer versions, but it'd be great if you can always keep the readme up to date.

**Comment by kwsim539** [comment]
Agreed. They should at the least confirm the code has been tested to the latest version.
